src/provider/camunda-platform/properties/Error.js

```diff
@@ -0,0 +1,280 @@
+import TextField, { isEdited as textFieldIsEdited } from '@bpmn-io/properties-panel/lib/components/entries/TextField';
+import { isEdited as selectIsEdited } from '@bpmn-io/properties-panel/lib/components/entries/Select';
+
+import ReferenceSelect from '../../../entries/ReferenceSelect';
+
+import {
+  useService
+} from '../../../hooks';
+
+import {
+  sortBy
+} from 'min-dash';
+
+import { getBusinessObject } from 'bpmn-js/lib/util/ModelUtil';
+
+import {
+  createElement,
+  findRootElementById,
+  findRootElementsByType,
+  getRoot,
+  nextId
+} from '../../../utils/ElementUtil';
+
+export const EMPTY_OPTION = '';
+export const CREATE_NEW_OPTION = 'create-new';
+
+
+export default function Error(props) {
+
+  const {
+    idPrefix,
+    element,
+    errorEventDefinition
+  } = props;
+
+  let entries = [{
+    id: idPrefix + '-expression',
+    component: <Expression element={ element } errorEventDefinition={ errorEventDefinition } idPrefix={ idPrefix } />
+  }, {
+    id: idPrefix + '-errorRef',
+    component: <ErrorRef element={ element } errorEventDefinition={ errorEventDefinition } idPrefix={ idPrefix } />,
+    isEdited: selectIsEdited
+  }];
+
+  const error = errorEventDefinition.get('errorRef');
+
+  if (error) {
+    entries = [
+      ...entries,
+      {
+        id: idPrefix + '-errorName',
+        component: <ErrorName element={ element } error={ error } idPrefix={ idPrefix } />,
+        isEdited: textFieldIsEdited
+      },
+      {
+        id: idPrefix + '-errorCode',
+        component: <ErrorCode element={ element } error={ error } idPrefix={ idPrefix } />,
+        isEdited: textFieldIsEdited
+      }
+    ];
+  }
+
+  return entries;
+}
+
+function Expression(props) {
+  const {
+    element,
+    errorEventDefinition,
+    idPrefix
+  } = props;
+
+  const commandStack = useService('commandStack');
+  const translate = useService('translate');
+  const debounce = useService('debounceInput');
+
+  const setValue = (value) => {
+    commandStack.execute('properties-panel.update-businessobject', {
+      element,
+      businessObject: errorEventDefinition,
+      properties: {
+        'camunda:expression': value
+      }
+    });
+  };
+
+  const getValue = () => {
+    return errorEventDefinition.get('camunda:expression');
+  };
+
+  return TextField({
+    element: errorEventDefinition,
+    id: idPrefix + '-expression',
+    label: translate('Throw Expression'),
```

Comment:
Sentence case should be the new standard
```suggestion
    label: translate('Throw expression'),
```

Reply:
Done (cf. https://github.com/bpmn-io/bpmn-properties-panel/pull/108/files#diff-e9ae2a1607ddb924b0430102a13f4e6591870d01ce2777ba90d86e146ea9c816R94).